package.json

```diff
@@ -14,7 +14,9 @@
   "version": "0.0.44",
   "author": "Alan K <ack@modeswitch.org> (http://blog.modeswitch.org)",
   "homepage": "http://filerjs.github.io/filer",
-  "bugs": "https://github.com/filerjs/filer/issues",
+  "bugs": {
```

Comment:
Although you added the package.json, I'm guessing it was pushed when you added everything and pushed it, remember to use the command git status to see the files that were changed and use git add filepath/file to add it singularily.

Reply:
Sorry I'm not familiar with Git, but by this do you mean I should only push <b>fs.open.spec.js</b> and discard changes for <b>package.json</b>?

Reply:
Exactly @struong9, you can revert this change to `package.json` with `git checkout master package.json` and then `git add` and `git commit` and `git push`